shell.nix

```diff
@@ -11,12 +11,10 @@ mkShell {
 
   shellHook = ''
     rm -f conductor-config.toml
+    rm -f src/utils/integration-testing/conductor-config.toml
     ln -s ${project.hp-admin-conductor-config} conductor-config.toml
-    echo "Starting Conductor..."
-    holochain -c conductor-config.toml &> conductor.log &
```

Comment:
Basically to avoid any side effects of opening Nix and to allow each of our script commands to manage the environment equally and cleanly.  Each test in the integration suite already starts and closes it's own environment (...and this also avoids spending unnecessary resources to spin up a conductor for to immediately close it and restart it with the first integration test.) :)